src/components/WorkMetadata/WorkMetadata.tsx

```diff
@@ -336,6 +338,30 @@ const WorkMetadata: React.FunctionComponent<Props> = ({
     <Tooltip id="tooltipLanguage">The primary language of the content.</Tooltip>
   )
 
+  const tooltipDownload = (
```

Comment:
Maybe this could be refactored either into it's own component or just make it a function accepting the id and and text.

Reply:
I will look into refactoring this at a later time. There is too much redundancy, but there are more than two variables that change.